pypck/request_handlers.py

```diff
@@ -355,13 +355,13 @@ def __init__(
         self,
         addr_conn: ModuleConnection,
         num_tries: int = 3,
-        timeout_msec: int = 1500,
+        timeout: float = 1500,
```

Comment:
hey, didn't you miss to update the value here?

Reply:
Yes, thanks. #131 will fix this.